aimmo-game/service.py

```diff
@@ -4,9 +4,8 @@
 import sys
 
 # If we monkey patch during testing then Django fails to create a DB enironment
```

Comment:
Comment is now invalid (tests are run separately to the Django tests now so there are no issues).